packages/flutter/test/widgets/scrollable_test.dart

```diff
@@ -267,4 +270,85 @@ void main() {
 
     expect(getScrollOffset(tester), 20.0);
   });
+
+  testWidgets('Vertical scrollables are scrolled when activated via keyboard.', (WidgetTester tester) async {
```

Comment:
Can you add more tests to ensure that it scrolls in the right direction when "reverse: true" and when a "center" sliver is used?